net.sourceforge.lgames.LGeneral.yaml

```
@@ -0,0 +1,37 @@
+app-id: net.sourceforge.lgames.LGeneral
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+rename-desktop-file: lgeneral.desktop
+rename-icon: lgeneral
+command: lgeneral
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --persist=.lgames
+
+modules:
+  - shared-modules/glu/glu-9.json
+  - shared-modules/SDL/SDL-1.2.15.json
+  - shared-modules/SDL/SDL_mixer-1.2.12.json
+  - name: lgeneral
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/lgeneral/lgeneral-1.4.4.tar.gz
+        sha256: 0a26b495716cdcab63b49a294ba31649bc0abe74ce0df48276e52f4a6f323a95
+      - type: file
+        filename: kukgen-data-1.1.tar.gz
+        url: http://downloads.sourceforge.net/lgeneral/kukgen-data-1.1.tar.gz
+        sha256: fbad68e6ec92325ff63dbe6ae77337f9806a4347667c712f94265419b89d6d84
+      - type: file
+        path: net.sourceforge.lgames.LGeneral.metainfo.xml
+      - type: file
+        path: lgeneral128.png
```

```suggestion
        path: lgeneral128.png
      - type: shell
        commands:
          - cp -p /usr/share/automake-*/config.{sub,guess} .
```

You need something like that to build. (the `.` assume `config.sub` is in the top level and must be adjusted accordingly)

tl;dr: the tarball is so old that the configure bootstrap doesn't know aarch64.